jacobian_follower/src/jacobian_follower/jacobian_follower.cpp

```diff
@@ -288,14 +290,22 @@ std::optional<moveit_msgs::RobotState> JacobianFollower::computeCollisionFreePoi
   moveit::core::GroupStateValidityCallbackFn constraint_fn_boost;
   constraint_fn_boost = boost::bind(&isStateValid, planning_scene, _1, _2, _3);
 
-  bool ok = robot_state_ik.setFromIK(joint_model_group,              // joints to be used for IK
-                                     EigenSTL::vector_Isometry3d(),  // this isn't used, goals are described in opts
-                                     std::vector<std::string>(),     // names of the end-effector links
-                                     0,                              // take values from YAML
-                                     constraint_fn_boost,
-                                     opts  // mostly empty
-  );
-  ROS_DEBUG_STREAM_NAMED(LOGGER_NAME + ".ik", "ok? " << ok);
+  bool ok = false;
+  auto attempts{0};
+  for (; attempts < 100; ++attempts) {
```

Comment:
`#define BIOIK_HARDCODED_ATTEMPTS 100`

Reply:
Obviously I don't love this hardcoded param, but seems acceptable for your current usage